src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java

```diff
@@ -49,10 +50,8 @@ protected void starting(Description description) {
     };
     
 
-    private SecurityDataResult processTestHelper( String cve_id) throws Exception
+    private SecurityDataResult processTestHelper( String cpe, String cve_id) throws Exception
     {
-        String cpe="cpe:/a:redhat:openshift_application_runtimes:1.0";
-
         SecurityDataProcessor sdp = new SecurityDataProcessor(cpe);
 
         SecurityDataResult sdpr = sdp.process( cve_id);
```

Comment:
Not for now, but longer term we may want to consider mocking the server ; the data api went down yesterday which meant the tests started failing!

Reply:
agreed. will do later.